src/@batch-flask/electron/electron-app.service.ts

```diff
@@ -16,7 +16,7 @@ export class ElectronApp {
 
     }
 
-    public getPath(name: string): string {
+    public getPath(name: any): string {
```

Comment:
Why any?

Reply:
Line 20 `getPath()` was running into an error when `name` was set to type `string` (even though its signature wanted a `string`) so I changed it to `any` for now.

Reply:
Clunky though it is, we should adhere to the string literal union type they're using as an enum:

```
public getPath(name:  'home' | 'appData' | 'userData' | 'cache' | 'temp' | 'exe' | 'module' | 'desktop' | 'documents' | 'downloads' | 'music' | 'pictures' | 'videos' | 'logs' | 'pepperFlashSystemPlugin'): string {
```

Otherwise, if someone passes something else in, we'll get an error at run-time.